.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,28 @@
-<!-- PLEASE ENSURE YOU HAVE READ CLOUDNET'S CODESTYLE GUIDELINES -->
-<!-- IF YOU'RE NOT FOLLOWING CLOUDNET'S CODESTYLE GUIDELINES, THEN THIS PULL REQUEST IS LIKELY TO BE REJECTED -->
-<!-- IF YOU'RE NOT PROVIDING ANY INFORMATION, THEN THIS PULL REQUEST IS LIKELY TO BE REJECTED -->
-
-This pull request includes:
-
-- [ ] breaking changes
-- [ ] no breaking changes
+<!-- 
+Thanks for taking your time and creating a pull request. Please note that we will not merge pull requests
+which are not following our code style (https://google.github.io/styleguide/javaguide.html). Most of these
+rules are checked while compile using checkstyle. On the other hand, please cover relevant code with tests.
+These are showing the maintainers what to expect from your pull requests and ensures that changes to your
+code will be consistent over time. See for example https://betterprogramming.pub/13-tips-for-writing-useful-unit-tests-ca20706b5368
+if you need a bit of guidance while writing your tests.
+-->
+
+### This pull request includes:
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing api/functionality to change)
+- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document.
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
```

Comment:
Are we talking about javadocs or [these ones](https://cloudnetservice.eu/docs/3.4/)?